docs/topics/js/js-overview.md

```diff
@@ -59,6 +59,61 @@ at the expense of type safety.
 Kotlin/JS is also compatible with the most common module systems: UMD, CommonJS, and AMD. Being able to [produce and consume modules](js-modules.md)
 means that you can interact with the JavaScript ecosystem in a structured manner.
 
+## Kotlin/JS frameworks
+
+Modern web development benefits significantly from frameworks that simplify building web applications.
+Here are examples of popular web frameworks for Kotlin/JS written by different authors:
+
+### KVision
+
+_KVision_ is an object-oriented web framework that makes it possible to write applications in Kotlin/JS without having to
+write HTML or CSS. Instead, it provides ready-to-use components that can be used as building blocks for your application’s
+user interface. You can use both reactive and imperative programming models to build your frontend, use connectors for Ktor,
+Spring Boot, and other frameworks to integrate it with your server-side applications, and share code using Kotlin Multiplatform.
+
+Visit [https://kvision.io](https://kvision.io) for documentation, tutorials, and examples.
+
+For updates and discussions about the framework, join [#kvision](https://kotlinlang.slack.com/messages/kvision) and
```

Comment:
We can make these descriptions about channels more consistent.

Also, don't you think that it's better to use the "from goal to action" way?
`To get the result, do this`. I mean, before we ask users to join the channels, we should explain the benefits (why they should do that). This sentence is OK, but others use different word order.